src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -47,12 +45,9 @@ uct_rc_mlx5_txqp_bcopy_post(uct_rc_mlx5_iface_common_t *iface,
                             uct_rc_iface_send_desc_t *desc, const void *buffer,
                             uct_ib_log_sge_t *log_sge)
 {
-    desc->super.sn = txwq->sw_pi;
-    uct_rc_mlx5_txqp_dptr_post(iface, IBV_QPT_RC, txqp, txwq,
-                               opcode, buffer, length, &desc->lkey,
-                               rdma_raddr, rdma_rkey, 0, 0, 0, 0,
-                               NULL, NULL, 0, fm_ce_se, imm_val_be, INT_MAX, log_sge);
-    uct_rc_txqp_add_send_op(txqp, &desc->super);
+    uct_mlx5_txqp_bcopy_post(iface, IBV_QPT_RC, txqp, txwq, opcode, length,
```

Comment:
no need for `uct_rc_mlx5_txqp_bcopy_post` now, can use `uct_mlx5_txqp_bcopy_post` directly